src/func.d

```diff
@@ -4486,7 +4486,7 @@ public:
         return this;
     }
 
-    override const(char)* kind()
+    override const(char)* kind() const
     {
         // GCC requires the (char*) casts
         return (tok != TOKfunction) ? cast(char*)"delegate" : cast(char*)"function";
```

Comment:
> I'm only doing it because I am stuck on working on MI because my PRs on it are twisting in the wind.

You're refactoring because you're being blocked on what you should really be doing?

Hmmmm... Maybe we should block you more often. :-)